strongbox-client/src/main/java/org/carlspring/strongbox/client/ArtifactClient.java

```diff
@@ -278,19 +295,26 @@ public void delete(String storageId,
                        boolean force)
             throws ArtifactOperationException
     {
-        @SuppressWarnings("ConstantConditions")
-        String url = getContextBaseUrl() + "/storages/" + storageId + "/" + repositoryId + "/" + path +
-                     (force ? "?force=" + force : "");
+        try
```

Comment:
@AlexOreshkevich : I don't think I quite agree with the `try`/`catch` here... Could you please remove it, unless there is a good reason I'm missing?